app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -41,10 +38,10 @@
           <% end %>
         </ul>
       <% end %>
-
       <% if card_helper.metadata %>
         <p class="gem-c-image-card__metadata"><%= card_helper.metadata %></p>
       <% end %>
     </div>
+    <%= card_helper.image %>
```

Comment:
I think this might need a bit more logic as it's broken the layout on mobile when there is no image.

Before | After
------ | ------
<img width="397" alt="Screenshot 2020-09-21 at 08 31 27" src="https://user-images.githubusercontent.com/861310/93741881-1795d780-fbe5-11ea-925c-a3e94329854a.png"> | <img width="385" alt="Screenshot 2020-09-21 at 08 31 32" src="https://user-images.githubusercontent.com/861310/93741899-1ebce580-fbe5-11ea-8fe4-457357a045e8.png">